playbooks/selfservice/selfservice-member-absent.yml

```diff
@@ -1,11 +1,11 @@
 ---
-- name: Delegation member absent
+- name: Selfservice member absent
   hosts: ipaserver
   become: true
 
   tasks:
-  - name: Ensure delegation "basic manager attributes" member attributes employeenumber and employeetype are absent
-    ipadelegation:
+  - name: Ensure selfservice "basic manager attributes" member attributes employeenumber and employeetype are absent
+    ipaselfservice:
       ipaadmin_password: SomeADMINpassword
       name: "basic manager attributes"
       attribute:
```

Comment:
Just a suggestion,
Instead of "employeenumber" and "employeetype", I think we can use "businesscategory" and "departmentnumber" attributes.